src/NServiceBus.Core/Encryption/RijndaelEncryptionService.cs

```diff
@@ -227,6 +227,11 @@ protected virtual bool TryGetKeyIdentifierHeader(out string keyIdentifier, IInco
             return context.Headers.TryGetValue(Headers.RijndaelKeyIdentifier, out keyIdentifier);
         }
 
+        protected virtual void GenerateIV(RijndaelManaged rijndael)
```

Comment:
Looks like a reasonable solution. @ramonsmits @danielmarbach are you OK with what @timbussmann proposes?